Orange/widgets/model/owrandomforest.py

```diff
@@ -27,7 +27,8 @@ class OWRandomForest(OWBaseLearner):
     index_output = settings.Setting(0)
 
     class Error(OWBaseLearner.Error):
-        not_enough_features = Msg("Insufficient number of attributes ({})")
+        not_enough_features = Msg(
+            'Number of considered attributes is too high (max {})!')
```

Comment:
I don't know who requested this change. It could have been me. I feel like an old professor. Anyway, I don't like it any better than the old message. I find it equally uninformative. :) 

What about `"Number of considered attributes exceeds the number of attributes"`? It's long, but the widget is probably wide enough.